go.mod

```diff
@@ -31,7 +31,10 @@ require (
 	gopkg.in/ini.v1 v1.67.0 // Apache 2.0
 )
 
-require gopkg.in/yaml.v3 v3.0.1
+require (
+	go.uber.org/mock v0.3.0
```

Comment:
This is Apache 2.0 licensed.

Reply:
Do we need to add this to our notice file?

Reply:
Good point! I added it. We could use some automation for this...